DEVELOPMENT.md

```diff
@@ -70,26 +84,33 @@ integration environments), update the
     jenv add /Library/Java/JavaVirtualMachines/adoptopenjdk-11.jdk/Contents/Home
     ```
 
-- The build and the integration tests assume `python3` and its built-in module `venv` are available. The
-  build of openapi creates a python virtual environment and activates it to run the script to merge
-  the various YAML files comprising the REST API.
+**NOTE**: You may encounter issues with the application when running an unexpected version of Java.
+So make sure you are running `AdoptOpenJDK Java 11 (Hotspot)` as specified above.
 
-  `python3` is typically available on *ix distributions, but may not be available Mac or Windows.
-  On Mac, you can install it using Homebrew like
-  ```
-  brew install python3
-  ```
-  The advantage of using `venv` is that it is built in so does not require extra installation steps.
-  However, it is not so useful as a developer tools because it has no way to de-activate a virtual
-  environment. Consider installing `virtualenv` for your developer use. It can be installed using
-  `pip` like so:
-  ```
-  python3 -m pip install virtualenv
-  ```
+#### Python3
+We are currently using Python 3.9.10
 
-- Recommended: read the [README](README.md) to understand the general structure of the service
+The build and the integration tests assume `python3` and its built-in module `venv` are available. The
+build of openapi creates a python virtual environment and activates it to run the script to merge
+the various YAML files comprising the REST API.
 
-**NOTE**: You may encounter issues with the application when running an unexpected version of Java. So make sure you are running `AdoptOpenJDK Java 11 (Hotspot)` as specified above.
+The recommended practice is to install `pyenv` and use that to install and manage your versions
+of python. Follow the instructions [here](https://github.com/pyenv/pyenv#homebrew-in-macos) to
+install pyenv.
+
+Then use `pyenv` to install the right version of python like so:
+```
+pyenv install 3.9.10
+```
+
+In the automation, we use the `venv` to make virtual environments. The advantage of using `venv`
+is that it is built in so does not require extra installation steps.
+However, it is not so useful as a developer tools because it has no way to de-activate a virtual
+environment. Consider installing `virtualenv` for your developer use. It can be installed using
+`pip` like so:
+```
+python3 -m pip install virtualenv
+```
```

Comment:
You've just recommended three different virtual environment solutions for Python: `pyenv`, `venv`, and `virtualenv`. Can we not settle on one (even if it means adding cruft to the dockerfile)?

Reply:
`pyenv` is not a virtual environment solution.

The docker Broad has running on their hosted servers is deliberately minimal. My instructions from devops are to install what I need in the GHA. So I prefer using venv rather than adding another installation step.